app/src/main/res/layout/activity_login.xml

```diff
@@ -1,95 +1,104 @@
+                <com.google.android.material.textfield.TextInputLayout
+                    android:id="@+id/til_username"
+                    style="@style/TextInputLayout.StrokeWhite"
+                    android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:layout_width="match_parent"/>
+                    android:layout_marginBottom="@dimen/layout_padding_20dp"
+                    android:backgroundTint="@color/white"
+                    android:textColorHint="@color/white">
 
-            </com.google.android.material.textfield.TextInputLayout>
+                    <com.google.android.material.textfield.TextInputEditText
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:fontFamily="@font/productsans"
+                        android:hint="@string/username"
+                        android:inputType="text"
+                        android:textColor="@color/white"/>
 
-            <!-- Password Label -->
-            <com.google.android.material.textfield.TextInputLayout
-                android:id="@+id/til_password"
-                android:layout_height="wrap_content"
-                android:layout_marginBottom="@dimen/default_margin"
-                android:layout_marginTop="@dimen/default_margin"
-                android:layout_width="match_parent"
-                app:passwordToggleEnabled="true">
+                </com.google.android.material.textfield.TextInputLayout>
 
-                <EditText
-                    style="@style/Base.TextAppearance.AppCompat.Medium"
-                    android:drawableLeft="@drawable/ic_wrapped_lock_black_24dp"
-                    android:drawablePadding="@dimen/layout_padding_8dp"
-                    android:drawableStart="@drawable/ic_wrapped_lock_black_24dp"
-                    android:hint="@string/password"
-                    android:inputType="textPassword"
+                <com.google.android.material.textfield.TextInputLayout
+                    android:id="@+id/til_password"
+                    style="@style/TextInputLayout.StrokeWhite"
+                    android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:layout_width="match_parent"/>
-            </com.google.android.material.textfield.TextInputLayout>
+                    android:backgroundTint="@color/white"
+                    android:textColorHint="@color/white"
+                    app:passwordToggleEnabled="true"
+                    app:passwordToggleTint="@color/white">
+
+                    <com.google.android.material.textfield.TextInputEditText
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:fontFamily="@font/productsans"
```

Comment:
Is this font used in other mifos repos also where redesign work is completed?

Reply:
yes.